README.md

```diff
@@ -1,368 +1,257 @@
-  
+
+2. Run the generator with a simple "Hello World" example:
+
+  ```bash
+  rails generate react_on_rails:install --hello_world_example
   ```
-  //= require react_on_rails
+
+3. NPM install and build client bundle:
+
+  ```bash
+  cd client 
+  npm install
+  npm run build:client
   ```
-1. Expose your client globals like [this](https://github.com/shakacode/react-webpack-rails-tutorial/blob/537c985dc82faee333d80509343ca32a3965f9dd/client/app/startup/clientGlobals.jsx#L3):
-  
-   ```javascript
-   import App from './ClientApp';
-   window.App = App;
-   ```
-1. Put your client globals file as webpack entry points like [this](https://github.com/shakacode/react-webpack-rails-tutorial/blob/537c985dc82faee333d80509343ca32a3965f9dd/client/webpack.client.rails.config.js#L22). Similar pattern for server rendering.
-  
-  ```javascript
-  config.entry.app.push('./app/startup/clientGlobals');
+
+4. Start your Rails server:
+
+  ```bash
+  foreman start -f Procfile.dev
   ```
-1. See customization of configuration options below.  
 
-### Additional Steps For Server Rendering (option `prerender` shown below)
-See the next section for a sample webpack.server.rails.config.js.
-1. Expose your server globals like [this](https://github.com/shakacode/react-webpack-rails-tutorial/blob/537c985dc82faee333d80509343ca32a3965f9dd/client/app/startup/serverGlobals.jsx#L7)
-   
-   ```javascript
-   import App from './ServerApp';
-   global.App = App;
-   ```
-2. Make the server globals file an entry point in your webpack config, like [this](https://github.com/shakacode/react-webpack-rails-tutorial/blob/537c985dc82faee333d80509343ca32a3965f9dd/client/webpack.server.rails.config.js#L7)
-
-   ```javascript
-   entry: ['./app/startup/serverGlobals'],
-   ```
-3. Ensure the name of your ouput file (shown [here](https://github.com/shakacode/react-webpack-rails-tutorial/blob/537c985dc82faee333d80509343ca32a3965f9dd/client/webpack.server.rails.config.js#L9)) of your server bundle corresponds to the configuration of the gem. The default path is `app/assets/javascripts/generated`. See below for customization of configuration variables.
-4. Expose `React` in your webpack config, like [this](https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/client/webpack.server.rails.config.js#L23)
+## Generator Options
+Run `rails generate react_on_rails:install --help` for descriptions of all available options:
 
-```javascript
-      { test: require.resolve('react'), loader: 'expose?React' },
-      
-      // For React 0.14
-      { test: require.resolve('react-dom/server'), loader: 'expose?ReactDOMServer' }, // not in client one, only server
+```
+Usage:
+  rails generate react_on_rails:install [options]
+
+Options:
+  -d, [--redux], [--no-redux]                        # Setup Redux files
```

Comment:
@justin808 I was thinking that if we were ever to make it possible to generate only a webpack integration without React, then R should be for React. But maybe that's stupid. I can change it. What do you think?